KevS-SK2116-Updates-CDN-guide-and-Statistics-and-logs-guide-for-WebHosting (#7860)
ovh/docs: pages/index.md

            + [How to migrate a website from a web hosting plan to a VPS](web_cloud/web_hosting/migrate_website_to_vps)
            + [Speeding up your website with CDN](web_cloud/web_hosting/cdn_how_to_use_cdn)
            + [Accessing a website’s logs and statistics on a Web Hosting](web_cloud/web_hosting/logs_and_statistics)

>             + [Accessing a CDN’s logs and statistics on a Web Hosting](web_cloud/web_hosting/cdn_statistics_and_logs)

Reviewer: ```suggestion
            + [How to view CDN statistics and logs on a Web Hosting plan](web_cloud/web_hosting/cdn_statistics_and_logs)
```
